### ISSUE TITLE
How to set SK lang

### ISSUE THREAD

**Issue by camohub** [Markette/Gopay#55]
Hi, I would like to have SK lang in GoPay form, and as I see GoPay support SK language https://doc.gopay.com/en/?php#lang 
But in the Markete are allowed only CZ/EN. 
There is a method: 

````
public function setLang($lang)
    {
        if (!in_array($lang, $this->allowedLang)) {
            throw new InvalidArgumentException('Not supported language "' . $lang . '".');
        }
        $this->lang = $lang;

        return $this;
    }
````
which does not allow any other language. If I change the code ($allowedLang property), to support SK lang it works. In my opinion condition in setLang method is useless. Everybody can ensure the valid lang code and in other way it throws an exception (as it works also for now). Is it really necessary to check allowed langs which are hardly coded in AbstractService->allowedLang? 
Thanks.

**EDIT**: Not valid lang code falls to default EN, what is expectable and desirable, so that condition looks really useless.

**Comment by f3l1x** [comment]
Hi. Thank you for your time.

What about stay strict and append only missing languages?

-----

Language code	Language
CS	Czech
EN	English
SK	Slovak
DE	German
RU	Russian

**Comment by camohub** [comment]
It si your dicision. But what about ES, PL, HU, ................................ 
I think it is not necessary to be strict here. Fallback to default is common pattern and I thing Gopay will support it also in future. So setLang can be so simple as possible.

**Comment by f3l1x** [comment]
Hmm. Good point. But I still thing that strictness should be on library side. It's no difference for programmers, if they'll type properly values, otherwise it'll throw an exception. 

I think this way is more clean. Also it's more predictable, because you know what you send to server. Fallbacks are not straightforward solutions, especially not for the beginners.

Do you agree? Would you mind prepare a PR for it?

**Comment by camohub** [comment]
I don know how to make PR from GH fork. It is not so simple if I remember. But I am still against the strict mode here. Exception as exception. It is the same. If Gopay will throw an exception in future I believe it will be clean also for beginners. And every new language will need new release of this library. Strict mode is mainly about safety. This is not user friendly. 
THX.

**Comment by camohub** [comment]
So I did PR. Now it is on you.

**Comment by camohub** [comment]
And what about change the exception to EN fallback in this method?

**Comment by f3l1x** [comment]
@camohub Sorry, I would like to maintain it as closed (strict) as possible. So no, no fallback. But thank you for your time.

**Comment by f3l1x** [comment]
Closed via #57
